docs/developers-guide/versioning.md

```diff
@@ -41,23 +41,27 @@ Sometimes called a minor release, we issue point releases when we add bug fixes
 
 Sometimes called a patch release, we issue these hotfix releases to fix security issues in a timely manner, or to undo a horrific regression.
 
-## Other release terms
+## Release channels
 
-### The Gold Release
+Metabase maintains different release channels:
 
-The gold release is the first release of a new major version of Metabase. So for Metabase version 46, the gold releases would be:
+- **Stable**: The recommended version of Metabase to use in production. See the [list of stable releases](../releases.md).
+- **Beta**: Use the beta release to try out upstream features in a staging environment prior to a new major release. Beta releases should mostly be stable, but they can have bugs that we'll fix before shipping the gold release, which is the first stable version of a major release.
+- **Nightly**: Contains all the work from the master branch. It's the most up to date, but it's only recommended for experimental usage.
```

Comment:
Currently, the nightly tag points to the latest patch of the most recent release branch, not master. 🤔  We shouldn't publish this unless/until we make that change.